tests/infrared/13/infrared-openstack.sh

```diff
@@ -5,77 +5,120 @@ set -e
 #  VIRTHOST=my.big.hypervisor.net
```

Comment:
Skipping a few files here which I can see are part of the ceilometer metrics smoketesting and not really part of this PR